.travis.yml

```diff
@@ -99,68 +99,115 @@ geosx_pangea_build: &geosx_pangea_build
   - gcloud auth activate-service-account --key-file=${GEOSX_GCLOUD_KEY}
   - gsutil cp -a public-read ${GEOSX_BUNDLE} gs://geosx/Pangea2/
 
+draft_script: &draft_script
+  script:
+  - echo TRAVIS_BUILD_ID is $TRAVIS_BUILD_ID
+  - echo TRAVIS_JOB_ID is $TRAVIS_JOB_ID
+  - |
+    pr_number=$(curl -sS -H "Travis-API-Version: 3" -X \
+          GET https://api.travis-ci.com/build/$TRAVIS_BUILD_ID/jobs | \
+          jq -r '.jobs[] | select( (.stage.name == "checks") and
+          (.id == $TRAVIS_JOB_ID)) | .build.pull_request_number')
+  - echo $pr_number
```

Comment:
Hi, using `TRAVIS_PULL_REQUEST` (see https://docs.travis-ci.com/user/environment-variables/) will probably save you some `jq` time ;)

Reply:
Thanks!

Reply:
You're welcome!

Be careful, if I read the doc correctly, `TRAVIS_PULL_REQUEST` is `false` when not a PR.
The script would typically fail for standard branch builds.
We surely need to check the value before calling the service.